tools/clang/lib/SPIRV/ModuleBuilder.cpp

```diff
@@ -678,9 +678,14 @@ uint32_t ModuleBuilder::getImageType(uint32_t sampledType, spv::Dim dim,
   }
 
   if (dim == spv::Dim::Dim1D)
-    requireCapability(spv::Capability::Sampled1D);
+    if (sampled == 2u)
+      requireCapability(spv::Capability::Image1D);
+    else
```

Comment:
When there are `else`s together with multiple `if`s, it's better to use `{}` to group them clearly. Otherwise we may see strange and hard to spot bugs.

Reply:
I will do this in the next CL.